pxr/usd/sdf/testenv/testSdfPathParser.cpp

```diff
@@ -233,11 +232,27 @@ int main()
         NULL
     };
 
+    char const* goodUtf8[] = {
```

Comment:
Is the reason that this is "goodUtf8" because the changes that allow for UTF-8 change the class of valid prim identifiers to just the continue class?

Reply:
Yes, this is correct

Reply:
It might be helpful to leave a comment here noting it can be removed once the utf8 env setting is phased out.